bin/launch-cluster.sh

```diff
@@ -12,6 +12,11 @@ set -o pipefail
 if [ -z "${DCOS_CHANNEL}" ]; then
```

Comment:
Each project has its own snowflake launchers with no solution to this situation in sight, but I'm currently writing one to dcos-commons/tools to replace the one in dcos-tests. Could switch spark over to the dcos-commons/tools one once it's available.